src/main/java/nextstep/handler/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,61 @@
+package nextstep.handler;
+
+import nextstep.common.constant.ErrorCode;
+import nextstep.common.dto.ErrorResponse;
+import nextstep.favorite.exception.FavoriteException;
+import nextstep.line.exception.LineNotFoundException;
+import nextstep.member.exception.UnAuthorizedException;
+import nextstep.path.exception.PathException;
+import nextstep.section.exception.SectionException;
+import nextstep.station.exception.StationException;
+import nextstep.station.exception.StationNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
+
+    @ExceptionHandler(LineNotFoundException.class)
+    public ResponseEntity<ErrorResponse> LineNotFoundExceptionHandler(LineNotFoundException e) {
```

Comment:
메서드 이름은 관례에 따라 소문자로 시작하도록 작성해보면 어떨까요?
```suggestion
    public ResponseEntity<ErrorResponse> lineNotFoundExceptionHandler(LineNotFoundException e) {
```